openslides_votecollector/seating_plan.py

```diff
@@ -5,6 +5,28 @@ def setup_default_plan():
     """
     Adds a default seating plan.
     """
+    limit = 200
```

Comment:
Oh, this is a seating plan for a specific customer. I will restore the original plan.